save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/service/CommentService.kt

```diff
@@ -31,8 +31,8 @@ class CommentService(
      */
     @Transactional
     fun saveComment(comment: CommentDto, authentication: Authentication) {
-        val userId = authentication.userId()
-        val user = userRepository.getByIdOrNotFound(userId)
+        val userName = authentication.username()
```

Comment:
what's the point to change id to name?

Reply:
In the common module we do not have access to methods from the authentication service, so we cannot get the ID (authentication.userId()) and have to use the name